docs/manuals/studio/gui.md

```diff
@@ -1,25 +1,177 @@
 # Support for Taipy GUI Markdown syntax
 
-The Taipy Studio extension provides a specific kind of support for
-[IntelliSense](https://learn.microsoft.com/en-us/visualstudio/ide/using-intellisense)
-in the context of writing
-[Markdown text for Taipy GUI](../gui/pages.md#using-markdown).<br/>
-Taipy Studio will help you complete constructs needed to define visual elements
-and even propose code elements (such as variable or function names) that the
-element might use. In this regard, Taipy Studio provides a kind of code
-completion service, for the Taipy GUI Markdown syntax.
+The Taipy Studio extension leverages the Visual Studio Code text edition functionality
+to accelerate the definition of Taipy GUI pages with the
+[Markdown syntax](../gui/pages.md#using-markdown):
 
-This feature is activated when you enter Markdown content:
+- [IntelliSense](https://learn.microsoft.com/en-us/visualstudio/ide/using-intellisense)
+  can be used in the context of page definition.<br/>
+  This auto-completion feature gives you quick access to visual element types, their
+  specific properties, and even code variables in certain situations.<br/>
+  See the [Auto-completion](#auto-completion) section for details.
+- Diagnostics are provided in the Problems panel and the code itself, showing syntactic
+  problems in your page definition. Some of these problems can be quickly fixed using
+  Quick Fixes.<br/>
+  See the [Diagnostics](#diagnostics) section for details.
+- [Snippets](https://code.visualstudio.com/docs/editor/userdefinedsnippets) are
+  short pieces of code that can be inserted into your code with a few keystrokes.<br/>
+  The Taipy Studio extension comes with a few code snippets detailed in
+  the section on [Snippets](#snippets).
+- A command is available to generate a complete visual element definition in the
+  Markdown text.<br/>
+  See the [Element Generation](#diagnostics) section for details.
 
-- In a Markdown file (the filename extension must be `.md`);
-- In any string that appears in Python source code.
+The extension is activated when Visual Studio Code finds Markdown text:
 
-Beside the IntelliSense service, Taipy Studio also reports, in the *Problems*
-view, warnings and errors it finds in your Markdown content.
+- In a Markdown file (the filename extension must be `.md`) which is
+  part of the currently opened project;
+- In any string that appears in Python source code, part of the currently
```

Comment:
more accurately would be with triple quote strings

Reply:
Ah didn't pay attention. Thanks!